<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_Login</class>
 <widget class="QWidget" name="Page_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/login.png</normaloff>:/login.png</iconset>
  </property>
  <property name="toolTip">
   <string>这是登录窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Page_Login
{
	background-color: rgb(255, 255, 255);
}

QPushButton
{
    border-radius: 6px;/*圆角边框*/
    color: #fff;
    background-color: #1abc9c;
    font-size: 20px;
    font-family: &quot;华文行楷&quot;;
    width: 80;
    height: 40;
} 

/*登录界面按钮样式设置*/
QPushButton[login_btn=&quot;login&quot;]:hover { background-color: #FFAFBD; }
QPushButton[login_btn=&quot;login&quot;]:pressed { background-color:#85ef47; } 
/*登录界面头像设置*/
QLabel#lb_head { border-radius:25px; }

QLineEdit {
    border-radius: 6px;  /*圆角边框*/
    border: 2px solid #bdc3c7;

    min-height: 12;
    width: 150;
    height: 20;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lb_gif">
     <property name="text">
      <string>用代码放置gif图片的地方</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horiMain" stretch="1,1,0">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_head">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/headGraph.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridUser">
       <property name="verticalSpacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelName">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用 户 名</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineUser">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>32</height>
          </size>
         </property>
         <property name="maxLength">
          <number>18</number>
         </property>
         <property name="placeholderText">
          <string>请输入用户名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPassword">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密 码</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="linePassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>32</height>
          </size>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
         <property name="placeholderText">
          <string>输入密码</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horiButton">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="login_btn" stdset="0">
        <string>login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
       <property name="login_btn" stdset="0">
        <string>login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineUser</tabstop>
  <tabstop>linePassword</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>btnExit</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>Page_Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
